### ISSUE TITLE
WE masterful treachery doesn't affect proc

### ISSUE THREAD

**Issue by kyranria** [WarEmu/WarBugs#13321]
Issues should be unique. Check if someone else reported 
the issue first, and please don't report duplicates.
If its ToK related, please check the wiki before posting: 
https://wiki.returnofreckoning.com/index.php/Main_Page

Expected behavior and actual behavior:
masterful increase all damage done by 15% except proc kiss and witchbrew

**Comment by Torquemadra** [comment]
Correct, this is how procs work, same as WL proc pack assault doesnt get 15% from flanking or 25% from primal fury

**Comment by kyranria** [comment]
seems weird because flanking increase all damage even kiss/witchbrew and last time i had tried with bubble sham +15% damage or sorc +10% damage it also increased the damage of kiss/witchbrew

**Comment by Torquemadra** [comment]
you are correct, investigating

**Comment by Tazo83** [comment]
i can confirm this flanking did increase all damage from all abilities the other tactics i though they do but seems like they dont work same as flanking atm

**Comment by tlitschye** [comment]
Push
